src/Microsoft.OData.Core/SelectedPropertiesNode.cs

```diff
@@ -223,12 +263,13 @@ internal enum SelectionType
         /// <param name="selectQueryOption">The value of the $select query option.</param>
         /// <param name="structuredType">The structured type of this node.</param>
         /// <param name="edmModel">The Edm model.</param>
+        /// <param name="version"> The OData version. Default value is <see cref="ODataVersion.V4"/></param>
         /// <returns>A tree representation of the selected properties specified in the query option.</returns>
-        internal static SelectedPropertiesNode Create(string selectQueryOption, IEdmStructuredType structuredType, IEdmModel edmModel)
+        internal static SelectedPropertiesNode Create(string selectQueryOption, IEdmStructuredType structuredType, IEdmModel edmModel, ODataVersion? version = ODataVersion.V4)

```

Comment:
>ODataVersion? [](start = 135, length = 13)

It seems you don't need the nullable type. 

otherwise, it seems the default value "null" is better.